### PR TITLE
Change 'series' to 'edition' 

### PR DESCRIPTION
**Pull request by DrakoPensulo**
Change 'series_total' to 'edition_count' and 'series_number' to 'edition_number'

Make the changes consistent in the whole code.

### CODE REVIEW

**Comment by DrakoPensulo** [comment]
The file tests\wallet\nft_wallet\test_nft_wallet.py needs to be updated so that more checks are successful.

**Comment by paninaro** [comment]
This will require an update in the GUI

**Comment by paninaro** [comment]
Generally we try to avoid maintain RPC backward compatibility. This change updates the RPC responses for `nft_get_nfts` and `nft_get_info` such that the `series_total`/`series_number` fields are replaced by their edition counterparts. Since other community developers may be impacted by this, we could either maintain backward compatibility by making the `edition_*` fields additive, or informing the community devs of this breaking change. I'll leave it to @trepca and @wjblanke to weigh in.

**Review by wriches — CHANGES_REQUESTED** [review]

File: chia/cmds/wallet.py [hunk 1]
```diff
@@ -595,7 +595,7 @@ def nft_wallet_create_cmd(
     "-st", "--series-total", help="[DEPRECATED] NFT series total number", type=int, default=1, show_default=True
 )
 @click.option("-sn", "--series-number", help="[DEPRECATED] NFT series number", type=int, default=1, show_default=True)
-@click.option("-ec", "--edition-count", help="NFT edition count, defaults to 1", type=int)
+@click.option("-ec", "--edition-count", help="NFT edition total, defaults to 1", type=int)
```
[the remark this answers is not in the document]
Comment (wriches) [reply chia/cmds/wallet.py]: Done

File: chia/cmds/wallet.py [hunk 2]
```diff
@@ -647,15 +647,16 @@ def nft_mint_cmd(
         license_uris_list = []
     else:
         license_uris_list = [lu.strip() for lu in license_uris.split(",")]
+    edition_total = edition_count
```
[the remark this answers is not in the document]
Comment (wriches) [reply chia/cmds/wallet.py]: Done

File: chia/cmds/wallet.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jack60612) [reply chia/cmds/wallet.py]: ah i did not want to change more if possible

**Review by jack60612 — APPROVED** [review]
lgtm

**Comment by wallentx** [comment]
Hi @DrakoPensulo, thanks for the PR. I believe we had already gotten this resolved in our release branch, whose changes haven't made it into main just yet, but will do so soon with one of our checkpoint PRs.
https://github.com/Chia-Network/chia-blockchain/commit/af60a81c92cba51c650ad9a2fd2bf653efcbaf0e
I just wanted to verify with you that our changes had addressed all of the areas that were the intent of your PR before closing this. Even if we do close this one, I'd be happy to give you a shout-out in our coming release for your contribution effort.

**Comment by DrakoPensulo** [comment]
Hi @wallentx . The changes in https://github.com/Chia-Network/chia-blockchain/commit/af60a81c92cba51c650ad9a2fd2bf653efcbaf0e look good to me. That is what I intended. Thank you.

**Pull request closed without merging**